app/controllers/amr_heatmap_controller.rb

```diff
@@ -0,0 +1,39 @@
+class AmrHeatmapController < ApplicationController
+  include ReportHelper
+
+  before_action :admin_required
+
+  power :viewable_samples
```

Comment:
I don't think this line is necessary either, as it's already done in `power.rb`. It looks like we only use `power` statements in controllers if we want to set different scopes for different controllers, e.g. 
`power :projects, map: { EDIT_ACTIONS => :updatable_projects }, as: :projects_scope`

Reply:
Removed!